packages/plugin-e2e/src/index.ts

```diff
@@ -53,10 +53,6 @@ export { AppConfigPage } from './models/pages/AppConfigPage';
 export { PluginConfigPage } from './models/pages/PluginConfigPage';
 export { AppPage } from './models/pages/AppPage';
 
-// e2e-selectors
-export { Components, Pages, APIs, E2ESelectors } from './e2e-selectors/types';
-export { resolveSelectors } from './e2e-selectors/resolver';
-
```

Comment:
It's unfortunate that these were exported like this. However, they were (and will continue to be) exposed through the plugin-e2e Playwright api and that's how our docs [suggest](https://grafana.com/developers/plugin-tools/e2e-test-a-plugin/selecting-elements#the-selectors-fixture) to use them so I think it's fine to not call this out as a breaking change.

Reply:
Might be worth mentioning it in the release notes or change log just in case someone imported them directly from the package.